extension/app/ts/background/iconHandler.ts

```diff
@@ -62,7 +62,12 @@ export async function updateExtensionIcon(websiteTabConnections: WebsiteTabConne
 }
 
 export async function updateExtensionBadge() {
+	if ((await getIsConnected())?.isConnected === false) {
+		await setExtensionBadgeBackgroundColor({ color: '#f6685e' })
+		return await setExtensionBadgeText({ text: '!' })
```

Comment:
Consider a yellow warning icon instead of a red exclamation point.  I feel like we should reserve red exclamation points for "you are being attacked!"
```suggestion
		await setExtensionBadgeBackgroundColor({ color: 'yellow' })
		return await setExtensionBadgeText({ text: '⚠️' })
```

Reply:
![image](https://user-images.githubusercontent.com/13102010/233940963-175412bb-c52d-4f0d-a32c-acbc3bc43334.png)

that does not really work, but I agree with the yellow color:
![image](https://user-images.githubusercontent.com/13102010/233941175-8cefd634-7627-406f-8f9e-aaddc16e49fd.png)

I made the dialog box yellow too:
![image](https://user-images.githubusercontent.com/13102010/233941265-5994b8c5-588f-40f7-8305-8311bfd2cf0c.png)

Reply:
Can we do transparent color and set the text to `⚠️`?  I feel like that would look quite good.